src/plugins/jqplot.donutRenderer.js

```diff
@@ -450,7 +454,10 @@
                 labelelem.css({left: x, top: y});
             }
         }
-               
+        if (this.totalLabel) {
+            var totalLabel = $('<div class="jqplot-data-label" style="position:absolute">' + this._totalAmount + '</div>').insertAfter(plot.eventCanvas._elem);
```

Comment:
Would be better to not have the inline style for the absolute positioning and use a specific class name here. (ps. sorry for super late reply - have been busy)